app-tasks/Dockerfile

```diff
@@ -17,16 +17,13 @@ RUN set -ex \
     && buildDeps=' \
        python-dev \
     ' \
-    && sparkDeps=' \
-       openjdk-8-jre \
-    ' \
     && gdal=' \
        gdal-bin \
        libgdal1h \
        libgdal-dev \
     ' \
     && echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/backports.list \
```

Comment:
This looks like it could also be removed, but the file gets deleted later so it doesn't contribute to the size of the final image, and it reduces friction to testing out installs from backports, so I left it as-is.